src/response/records.ts

```diff
@@ -51,6 +51,12 @@ export interface Device {
 export interface Email {
   readonly firstSeen: string;
   readonly isFree?: boolean;
+
+  /**
+   * Deprecated effective August 29, 2019.
+   *
+   * @category Deprecated
+   */
   readonly isHighRisk?: boolean;
```

Comment:
Would it make sense to use `@deprecated` instead so that things like [tslint](https://github.com/palantir/tslint/pull/2395) warn?

Reply:
I'll add `@deprecated` in addition to `@category`. I wasn't thinking of TSLint when implementing this. I used `@category` because `TypeDoc` does not have a `@deprecated` tag.

Reply:
FWIW, Microsoft is [using it in vscode](https://github.com/microsoft/vscode/blob/37981d8fd5d4ba1619f69dc8a0f4670543d39650/src/vs/editor/common/modes/languageConfiguration.ts#L79).